src/Integration.Vsix/SonarLintNotificationsPackage.cs

```diff
@@ -46,40 +49,84 @@ public sealed class SonarLintNotificationsPackage : Package
         private const string NotificationDataKey = "NotificationEventData";
 
         private readonly IFormatter formatter = new BinaryFormatter();
-        private readonly NotificationIndicator notificationIcon = new NotificationIndicator();
+        private NotificationIndicator notificationIcon;
```

Comment:
The notification icon contains WPF elements so it has to be created on the UI thread.
The package is now loaded on a background thread so the field can't be created using a field initializer.